modules/efk-logging-deploy-playbook.adoc

```diff
@@ -36,9 +36,25 @@ from the registry which can take up to a few minutes.
 +
 ----
 $ oc get pods -w
+logging-curator-1541129400-l5h77           0/1       Running   0          11h  <1>
```

Comment:
The output are different with v3.x.  I guess we have to modify most of the output in v4.0.  For example: the fluentd pod name is like fluentd-xxxx rather than logging-fluentd-xxx. The pod name also changed for ES and kibana.  I will try to give you a snapshot once I have a v4.0 Env.